test/projectOverviewResponse.test.js

```diff
@@ -11,7 +11,7 @@ describe('Project Overview Response', () => {
     ];
     const expectedResult = smb()
       .attachment()
-      .text("*  *facebook* facebook.com\n*  *google* google.com")
+      .text('*  *facebook* facebook.com\n*  *google* google.com')
```

Comment:
Same as the other comment. Try to keep these linting fixes in a separate commit. For example, the 2 linting fixes could go in a commit with a message of: 'fix various linting errors"